tools/monitor/gui.py

```diff
@@ -0,0 +1,225 @@
+# coding=utf-8
+import curses
+import threading
+from colorama import Fore, Style, Back
+from time import sleep
+from enum import Enum
+
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../PWSat2OBC/integration_tests'))
+sys.path.append(os.path.join(os.path.dirname(__file__), '../..'))
```

Comment:
Jeżeli coś takiego moze się gdzieś znaleźć to tylko w pliku, który jest `main`em

Reply:
Usunięte.